src/js/views/eva-variant-view.js

```diff
@@ -335,7 +353,7 @@ EvaVariantView.prototype = {
                 variantInfo.chromosome = _this.getChromosomeNumberForAccessionWithRetries(response.data.contig);
                 variantInfo.start = response.data.start;
                 variantInfo.reference = response.data.referenceAllele;
-                if (response.data.alternateAllele) {
+                if (response.data.alternateAllele !== undefined && response.data.alternateAllele !== null) {
```

Comment:
what's the difference between `if(variable)` and `if(variable !== undefined && variable !== null)` ?
I have seen `if(variable)` in more places in this PR but don't know if those are ok or not...

Reply:
if (variable) is fine if we are just checking for empty variable (which is what we need to do in most places) but since empty string is allowed for alternate allele we need another way to check if it has invalid value.